File: patch.py
with open("packages/web-app-vercel/contexts/__tests__/PlaylistPlaybackContext.test.tsx", "r") as f:
    content = f.read()

fixed_content = content.replace("""<<<<<<< HEAD
    const result = generateShuffledIndices(-1);
=======
    const result = generateShuffledIndices(-5);
>>>>>>> origin/main""", "    const result = generateShuffledIndices(-1);")

with open("packages/web-app-vercel/contexts/__tests__/PlaylistPlaybackContext.test.tsx", "w") as f:
    f.write(fixed_content)

File: patch.js
const fs = require('fs');
const file = 'packages/web-app-vercel/components/ConfirmDialog.tsx';
let content = fs.readFileSync(file, 'utf8');

content = content.replace(
  /import \{ useState, useCallback \} from "react";/,
  `import { useState, useCallback } from "react";\nimport { Button } from "@/components/ui/button";`
);

content = content.replace(
  /<button[\s\S]*?onClick=\{onCancel\}[\s\S]*?>[\s\S]*?\{cancelText\}[\s\S]*?<\/button>/,
  `<Button variant="secondary" onClick={onCancel}>\n            {cancelText}\n          </Button>`
);

content = content.replace(
  /<button[\s\S]*?onClick=\{onConfirm\}[\s\S]*?>[\s\S]*?\{confirmText\}[\s\S]*?<\/button>/,
  `<Button variant={isDangerous ? "destructive" : "default"} onClick={onConfirm}>\n            {confirmText}\n          </Button>`
);

fs.writeFileSync(file, content);
